src/AppBundle/Security/AppVoter.php

```diff
@@ -123,8 +148,24 @@ private function canEdit(Lecture $subject, User $user)
      * @param User $user
      * @return bool
      */
-    private function canCreateOrDelete(User $user)
+    private function canCreateOrDelete(HasOwnerInterface $subject, User $user, TokenInterface $token)
     {
-        return $user->hasRole('ROLE_ADMIN');
+        $reflect = new \ReflectionClass($subject);
```

Comment:
Kokios dokumentacijos? Aš šitą darau, kad iš Config Managerio Entity configus išsitraukti, nes EasyAdminas vadina Entičius klasių vardais. Čia bet koks būdas tiks, kuris iš objekto klasės vardą leis išgaut.